### PR TITLE
Fix issues found in review

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
fixed some styling issues with the application.

### CODE REVIEW

**Review by srejitk — APPROVED** [review]
No code changes suggested. Small PR. Good Work!

File: src/components/ProductCard.jsx [hunk 1]
```diff
@@ -50,43 +53,47 @@ const ProductCard = ({
...
-              <span class="material-icons">favorite</span>
-            )}
-          </span>
-        </div>
-        <div className="ratings-container">
-          <span className="btn numbered-rating numbered-rating-position">
-            <span>{rating}</span>
-            <span className="material-icons">star</span>
-          </span>
+           { 1===2 ? "Order Now" : <Link to="/Cart">Go to cart</Link>}
```
Comment (nishasen) [inline src/components/ProductCard.jsx]: check this condition once

**Review by nishasen — COMMENTED** [review]
Everything looks good

**Review by nishasen — APPROVED** [review]

File: src/components/ProductCard.jsx (hunk 1 above)
Comment (vaibhav-mahalle) [reply src/components/ProductCard.jsx]: it was a placeholder condition, have made the changes

**Pull request merged**